charts/backstage/README.md.gotmpl

```diff
@@ -273,19 +273,25 @@ upstream:
 ## Installing RHDH with Orchestrator
 
 Orchestrator brings serverless workflows into Backstage, focusing on the journey for application migration to the cloud, on boarding developers ,and user-made workflows of Backstage actions or external systems.
-Orchestrator is a flavor of RHDH, and can be installed alongside the RHDH in the same namespace and in the folloing way: 
+Orchestrator is a flavor of RHDH, and can be installed alongside the RHDH in the same namespace and in the following way: 
 
-1. Have an admin install the orchestrator-infra helm chart, which will install the pre-requisites required to install RHDH flavored Orchestrator. This proccess will include installing cluster-wide resources, so should be done with admin privileges
+1. Have an admin install the [orchestrator-infra helm chart](https://github.com/redhat-developer/rhdh-chart/tree/main/charts/orchestrator-infra#readme), which will install the pre-requisites required to install RHDH flavored Orchestrator. This proccess will include installing cluster-wide resources, so should be done with admin privileges
 ```
-helm install <release_name> charts/orchestrator-infra
+helm install <release_name> rhdh-developer/orchestrator-infra
 ```
 2. Manually approve the Install Plans created by the chart, and wait for the Openshift Serverless and Openshift Serverless Logic Operators to be deployed. 
-3. Install backstage chart with helm, setting orchestrator to be enabled.
-4. Enable serverlessLogicOperator and serverlessOperator in the backstage values.
+3. Enable serverlessLogicOperator and serverlessOperator in the backstage values.
+4. Install backstage chart with helm, setting orchestrator to be enabled, like so:
```

Comment:
```suggestion
4. Install the `backstage` chart with Helm, enabling the orchestrator like so:
```

And do we need step 3, if the `helm install` command already enables "serverlessLogicOperator and serverlessOperator in the backstage values" ?